### PR TITLE
Create ui-c8y-1020-35-2-fix-typeahead-validation-when-using-json-path-in-display-property.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-3955](https://cumulocity.atlassian.net/browse/DM-3955)
Original PR: [7330](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7330)

[DM-3955]: https://cumulocity.atlassian.net/browse/DM-3955?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-35-2-fix-typeahead-validation-when-using-json-path-in-display-property.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fix typeahead validation when using JSON path in 'displayProperty'
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-35-2-fix-typeahead-validation-when-using-json-path-in-display-property.md]: The Claude AI suggested to use the following title:
```suggestion
title: Typeahead validation fixed when using JSON path in 'displayProperty'
```

File: content/change-logs/application-enablement/ui-c8y-1020-35-2-fix-typeahead-validation-when-using-json-path-in-display-property.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3955
+version: 1020.35.2
+---
+Fix typeahead validation when using JSON path in 'displayProperty'
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-35-2-fix-typeahead-validation-when-using-json-path-in-display-property.md]: The Claude AI suggested to use the following body:
```suggestion
Previously, when using a JSON path in the 'displayProperty' of a typeahead control, the validation was not working correctly and could lead to unexpected behavior. This issue has now been fixed. Typeahead controls using a JSON path in 'displayProperty' will now properly validate the entered values. This change improves the reliability and usability of typeahead controls in forms and applications where JSON paths are utilized.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1020-35-2-fix-typeahead-validation-when-using-json-path-in-display-property.md (hunk 1 above)
Comment (bozhidardedov) [inline content/change-logs/application-enablement/ui-c8y-1020-35-2-fix-typeahead-validation-when-using-json-path-in-display-property.md]:
```suggestion
title: Typeahead control validation fixed when using JSON path in 'displayProperty'
```

File: content/change-logs/application-enablement/ui-c8y-1020-35-2-fix-typeahead-validation-when-using-json-path-in-display-property.md (hunk 2 above)
Comment (bozhidardedov) [inline content/change-logs/application-enablement/ui-c8y-1020-35-2-fix-typeahead-validation-when-using-json-path-in-display-property.md]:
```suggestion
Previously, when using a JSON path in the 'displayProperty' of a typeahead control, the validation not allowing free entries (allowFreeEntries = false) was not working correctly and could lead to unexpected behavior, like creating orphan entries when an item needs to be assigned to a parent entity, etc. This issue has now been fixed. This change improves the reliability and usability of typeahead controls in forms.
```

File: content/change-logs/application-enablement/ui-c8y-1020-35-2-fix-typeahead-validation-when-using-json-path-in-display-property.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3955
+version: 1020.35.2
+---
+Previously, when using a JSON path in the 'displayProperty' of a typeahead control, the validation not allowing free entries (allowFreeEntries = false) was not working correctly and could lead to unexpected behavior, like creating orphan entries when an item needs to be assigned to a parent entity, etc. This issue has now been fixed. This change improves the reliability and usability of typeahead controls in forms.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-35-2-fix-typeahead-validation-when-using-json-path-in-display-property.md]:
```suggestion
Previously, when using a JSON path in the `displayProperty` of a typeahead control, the validation not allowing free entries (allowFreeEntries = false) was not working correctly and could lead to unexpected behavior, such as creating orphan entries when an item needs to be assigned to a parent entity. This issue has now been fixed. This change improves the reliability and usability of typeahead controls in forms.
```

**Review by janhommes — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**